manifest.json

```diff
@@ -53,7 +54,7 @@
 
 	"background":
 	{
-		"scripts": [ "js/background.js" ],
+		"scripts": [ "js/lib/lzutf8.min.js","js/background.js" ],
```

Comment:
Note : The order is important here, as the scripts are loaded in order. LZUTF8 is needed for background.js to work, so LZUTF8 has to be declared before background.js